public/data/imran.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Imran Khan",
+    "type": "personal",
+    "bio": "Open Source Enthusiast !",
+    "avatar": "https://github.com/YOUR_GITHUB_USERNAME.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/imran201khan",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/imrankhan",
+        "icon": "Linkden"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "DSA",
+        "date": "December 2021",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Started freelancing",
```

Comment:
```suggestion
        "description": "Started freelancing"
```

Reply:
I think the description should match the title..

Reply:
I've not changed the description, just removed the trailing comma.